src/html/ui/pane.html

```diff
@@ -46,6 +46,7 @@
                 <li class="crm-power-pane-subgroup" id="crm-diagnostics"><span><span class="icon crm-diagnostics-icon"></span>Crm Diagnostics</span></li>
                 <li class="crm-power-pane-subgroup" id="performance-center"><span><span class="icon performance-center-icon"></span>Performance Center</span></li>
                 <li class="crm-power-pane-subgroup" id="mobile-client"><span><span class="icon mobile-icon"></span>Mobile Client</span></li>
+				<li class="crm-power-pane-subgroup" id="open-webapi"><span><span class="icon record-url-icon"></span>Open Web API Record</span></li>
```

Comment:
Again, the indentation seems a bit off on the browser, not sure if it's due to editor preferences.

Reply:
Had to fix using Notepad. Not sure what Notepad++ or VS Code is showing correctly. Thanks for the feedback.